added readme, license and contribute guidelines (#2)
cachethq/docs: CONTRIBUTE.md

> # Contribute

Reviewer: I think this file should live within `.github/CONTRIBUTING.md`. This makes the contributing guide discoverable by GitHub.

For example, https://github.com/cachethq/cachet/blob/2.4/.github/CONTRIBUTING.md

Author: whops, very true.

## Introduction

> First, thank you for considering contributing to cachet! It's people like you that make the open source community such a great community! 😊

Reviewer: ```suggestion
First, thank you for considering contributing to Cachet! It's people like you that make the open source community such a great community! 😊
```

## Your First Contribution

Working on your first Pull Request ever? You can learn how from this _free_ series, [How to Contribute to an Open Source Project on GitHub](https://app.egghead.io/playlists/how-to-contribute-to-an-open-source-project-on-github).

Not too familiar with markdown? Have a look at this also free [Markdown tutorial](https://www.markdowntutorial.com/) to get started or refresh your memory.

## Submitting code

Any code change should be submitted as a pull request. The description should explain what the code does and give steps to execute it. The pull request should also contain tests.

## Code review process

The bigger the pull request, the longer it will take to review and merge. Try to break down large pull requests in smaller chunks that are easier to review and merge.
It is also always helpful to have some context for your pull request. What was the purpose? Why does it matter to you?

## Security Vulnerabilities

Please review our [security policy](https://github.com/cachethq/cachet/security/policy) on how to report security vulnerabilities.

## Questions

> If you have any questions, create an [issue](issue) (protip: do a quick search first to see if someone else didn't ask the same question before!).

Reviewer: ```suggestion
If you have any questions, please create a new [issue](https://github.com/cachethq/docs/issues).
```
